http_test.go

```diff
@@ -1372,7 +1372,7 @@ func TestResponseSuccess(t *testing.T) {
 
 	// response with missing server
 	testResponseSuccess(t, 500, "aaa", "", "aaadfsd",
-		500, "aaa", string(defaultServerName))
+		500, "aaa", "")
```

Comment:
Because the logic to handle defaultServerName is in the server code. This test just dumps a `Response` object to a buffer and parses it again. I think this new way is more correct. When I make a custom Response object and write it somewhere I don't expect fasthttp to just add the Server header if I didn't set it.